src/Modules/CalcOffence.lua

```diff
@@ -1549,41 +1549,32 @@ function calcs.offence(env, actor, activeSkill)
 			output.TriggerTime = output.Time
 			output.Speed = 1 / output.Time
 		elseif skillData.triggerRate and skillData.triggered then
-			-- Account for trigger unleash
-			if skillData.triggerUnleash then
-				-- process the source trigger skill to get it's full data
-				local calcMode = env.mode == "CALCS" and "CALCS" or "MAIN"
-				for _, triggerSkill in ipairs(actor.activeSkillList) do
-					if cacheSkillUUID(triggerSkill) == skillData.triggerSourceUUID then
-						calcs.buildActiveSkill(env, calcMode, triggerSkill)
-						break
-					end
-				end
-				local cachedSourceSkill = GlobalCache.cachedData[calcMode][skillData.triggerSourceUUID]
-				-- if properly processed, get it's dpsMultiplier to increase triggerRate
-				if cachedSourceSkill then
-					skillData.unleashTriggerRate = skillData.triggerRate * (cachedSourceSkill.ActiveSkill.skillData.dpsMultiplier or 1)
-					if breakdown then
-						breakdown.Speed = {
-							s_format("%.2f ^8(trigger rate)", skillData.triggerRate),
-							s_format("* %.2f ^8(multiplier from Unleash)", cachedSourceSkill.ActiveSkill.skillData.dpsMultiplier or 1),
-							s_format("= %.2f", skillData.unleashTriggerRate),
-						}
-					end
-					-- over-write the triggerRate modifier after breakdown as other calcs use it
-					skillData.triggerRate = skillData.unleashTriggerRate
-				end
-				-- give this activeSkill "HasSeals" flag so Configuration Option for UseMaxUnleash is available
-				activeSkill.skillFlags.HasSeals = true
-			end
 			output.Time = 1 / skillData.triggerRate
 			output.TriggerTime = output.Time
 			output.Speed = skillData.triggerRate
 			skillData.showAverage = false
 		elseif skillData.triggeredByBrand and skillData.triggered then
-			output.Time = 1 / (1 + skillModList:Sum("INC", cfg, "Speed", "BrandActivationFrequency") / 100) / skillModList:More(cfg, "BrandActivationFrequency") * (skillModList:Sum("BASE", cfg, "ArcanistSpellsLinked") or 1)
+			local activationFreqInc = 100 / (100 + skillModList:Sum("INC", cfg, "Speed", "BrandActivationFrequency"))
+			local activationFreqMore = skillModList:More(cfg, "BrandActivationFrequency")
+			local linkedSpellsCount = skillModList:Sum("BASE", cfg, "ArcanistSpellsLinked") or 1
+			
+			output.Speed = activationFreqMore * linkedSpellsCount * activationFreqInc
+			output.Time = 1 / output.Speed
```

Comment:
```suggestion
			local activationFreqInc = (100 + skillModList:Sum("INC", cfg, "Speed", "BrandActivationFrequency")) / 100
			local activationFreqMore = skillModList:More(cfg, "BrandActivationFrequency")
			local linkedSpellsCount = skillModList:Sum("BASE", cfg, "ArcanistSpellsLinked") or 1
			
			output.Speed = activationFreqInc * activationFreqMore / linkedSpellsCount
			output.Time = 1 / output.Speed
```
This is how it should be according to the original implementation.

Reply:
Applied [d1bfd70](https://github.com/PathOfBuildingCommunity/PathOfBuilding/pull/4599/commits/d1bfd705c58102fe6144e123de4162a4b07e0fc8). Modified breakdown to match. Thanks!